src/file.rs

```diff
@@ -49,10 +49,7 @@ impl Ec for EcFile {
     }
 
     fn version(&mut self) -> String {
-        let mut version = unsafe { self.get_str(b"VER:") };
-        while version.chars().next() == Some(' ') {
-            version.remove(0);
-        }
-        version
+        let version = unsafe { self.get_str(b"VER:") };
+        String::from(version.trim_start_matches(' '))
```

Comment:
This allocates twice, compared to the original code which only allocates a string once, then shifts the bytes around as necessary.